### DISCUSSION TITLE
[🐛] Botones no aparecen: whatsapp-web.js

### DISCUSSION THREAD

**Discussion by Aronchavez** [codigoencasa/builderbot#316] [Q&A] [answered]
Alguien pudo activar los botones, lo del multidispositivo ya no se puede desactivar de parte de WhatsApp lograron activarlos si me puede ayudar

**Comment by juan-silveira** [comment]
Como activar los botones? Estoy solo con uno dispositivo y no funciona.

**Comment by KJoaquin** [comment, marked as answer]
npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list

**Comment by leifermendez** [comment]
Exactamente como dice @KJoaquin

**Comment by VMASPAD** [comment]
no esta funcionando 28/4
